hpOneView/resources/servers/server_hardware.py

```diff
@@ -174,6 +174,27 @@ def add(self, information, timeout=-1):
         """
         return self._client.create(information, timeout=timeout)
 
+    def add_multiple(self, information, timeout=-1):
```

Comment:
Since the name of the method is diverging from the name of the endpoint, perhaps it is worth talking to the team to make sure this is being easily understandable.

Both 'discovery' and 'add_multiple' are overall generic, while the action it performs is not so generic, it is specific to adding multiple rack-mounted servers.

I say this because when you add in an enclosure frame link, you also add in multiple servers, meaning all the ones connected to that enclosure, and this was the first thing that came to mind when I saw the method name.

Reply:
As discussed, Keeping it as is.

Reply:
I think, we should change the function name to make the code more readable.
-> "add_multiple_servers"

Reply:
Kept it as `add_multiple` as the previous function `add` added a single server. So this name looked obvious for multiple servers. Anyways changed it to `add_multiple_servers` now.